CMakeLists.txt

```
@@ -277,53 +237,19 @@ if (ARMA_EXTRA_DEBUG)
   add_definitions(-DARMA_EXTRA_DEBUG)
 endif()
 
```

Here we can translate from the names in our `option()`s to what's needed by `find_mlpack()` or `fetch_mlpack()`:

```suggestion
if (NOT USE_OPENMP)
  set(MLPACK_DISABLE_OPENMP ON)
endif ()

if (USE_SYSTEM_STB)
  set(MLPACK_USE_SYSTEM_STB ON)
endif ()
```